java/README.md

```diff
@@ -10,7 +10,7 @@ library for producing output in JSON format.
     This example was provided by an early instructor user of the autograder
     platform and is not actively maintained. It may not support all current
     autograder platform functionality. See our
-    [Community Resources page](/resources/) for alternative Java resources.
+    [Community Resources page](https://github.com/gradescope/autograder_samples/blob/master/docs/resources.md) for alternative Java resources.
```

Comment:
I think it'd be better to link to ReadTheDocs, since that's the primary intended way to read the autograder docs. (The GitHub repo tries to be readable, and I tried to make the navigation work in a way that relative links would work on RTD and on GitHub, but I found that that doesn't really work well so we prefer making direct links to RTD).

```suggestion
    [Community Resources page](https://gradescope-autograders.readthedocs.io/en/latest/resources/) for alternative Java resources.
```